msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -229,7 +229,7 @@ Anonymous users can view and interact with Adaptive Cards in the meeting chat. A
 
 ## Known issues and limitations
 
-* For an anonymous user, the user ID from `getContext` and the user ID received by the bot are different. It's not possible to correlate the two directly. If you need to track the user's identity between your tab and bot, you must prompt the user to authenticate with an external identity provider.
+* Currently, the `getContext` API doesn't return a user ID for the anonymous user though the bot APIs do and it's not possible to correlate the anonymous user between these two APIs. If you need to track the user's identity between your tab and bot, you must prompt the user to authenticate with an external identity provider.
```

Comment:
Please remove "If you need to track the user's identity between your tab and bot, you must prompt the user to authenticate with an external identity provider."